src/zopflipng/zopflipng_lib.cc

```diff
@@ -423,3 +426,64 @@ int ZopfliPNGOptimize(const std::vector<unsigned char>& origpng,
 
   return error;
 }
+
+extern "C" void CZopfliPNGSetDefaults(CZopfliPNGOptions *png_options) {
+    
+    memset(png_options, 0, sizeof(*png_options));
+    // Constructor sets the defaults
+    ZopfliPNGOptions opts;
+
+    png_options->lossy_transparent    = opts.lossy_transparent;
+    png_options->lossy_8bit           = opts.lossy_8bit;
+    png_options->auto_filter_strategy = opts.auto_filter_strategy;
+    png_options->use_zopfli           = opts.use_zopfli;
+    png_options->num_iterations       = opts.num_iterations;
+    png_options->num_iterations_large = opts.num_iterations_large;
+    png_options->block_split_strategy = opts.block_split_strategy;
+}
+
+extern "C" int CZopfliPNGOptimize(const unsigned char *origpng,
```

Comment:
Please put the asterix's next to the type name rather than the variable name, to have the same style as the other code in this file